### PR TITLE
Fixed customization for Hive/Oozie metastores

### PR DESCRIPTION
**Pull request by wawon-msft**
## Description

There was a bug that would occur if a user specified an Hive/Oozie metastore when they had configurations like hive-site. This change will fix this scenario so the configurations are merged, but still raise an exception when there are duplicate configurations. There is no regeneration necessary for this change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
 		<Description>Azure HDInsight Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
-		<Version>3.1.0-preview</Version>
+		<Version>3.1.1-preview</Version>
 		<PackageTags>Microsoft Azure HDInsight Management;HDInsight;HDInsight Management</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
```
Comment (dsgouda) [inline src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj]: Please update `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks OK apart  from the comment.

File: src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj (hunk 1 above)
Comment (wawon-msft) [reply src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj]: Good catch.

**Comment by wawon-msft** [comment]
Thanks for the review. I've resolved the comment.

**Review by dsgouda — APPROVED** [review]
LGTM subject to CIs passing

**Review by dsgouda — CHANGES_REQUESTED** [review]
@wawon-msft  is planning to introduce more changes

**Comment by wawon-msft** [comment]
**@dsgouda**: We've decided to delay the other changes until later. You can merge these current changes.

**Review by dsgouda — APPROVED** [review]

**Pull request merged**